docs/pages/admin-guides/access-controls/sso/azuread.mdx

```diff
@@ -129,7 +129,7 @@ kind: saml
 metadata:
   name: ad
 spec:
-  acs: https://mytenant.teleport.sh/v1/webapi/saml/acs/ad
+  acs: example.teleport.sh/v1/webapi/saml/acs/ad
```

Comment:
I'm not sure how correct it is to get rid of `https://` here. We should probably keep it and only change `mytenant` to `example`.